### PR TITLE
Componets

### PR DESCRIPTION
**Pull request by vigehi**
Added clculator UI and its comonents

### CODE REVIEW

File: src/components/calc.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <button type="button" className="btn13">-</button>
+        <button type="button" className="btn14">1</button>
+        <button type="button" className="btn15">2</button>
+        <button type="button" className="btn16">3</button>
+        <button type="button" className="btn17">+</button>
+        <button type="button" className="btn18">0</button>
+        <button type="button" className="btn19">.</button>
+        <button type="button" className="btn20">=</button>
+      </div>
+    </div>
```
Comment (mikethreels) [inline src/components/calc.js]: - [x] the `calc` component should be class based according to the requirements

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Edith, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
- professional looking repo

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calc.js (hunk 1 above)
Comment (vigehi) [reply src/components/calc.js]: great reiew thanks

Comment (mahbubul14) [inline src/components/calc.js]: - [x] As the previous reviewer suggested, the `calc` component should be class-based according to the requirements. Kindly work on it. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @vigehi ,

You've done well so far, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive Readme file.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [x] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/calc.js (hunk 1 above)
Comment (vigehi) [reply src/components/calc.js]: i dont understan what you are saying

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @vigehi ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**